src/containers/table-container-row/table-container-row.js

```diff
@@ -19,9 +20,9 @@ const TableContainerRow = ({
 
   const dense = useSelector(({ Table }) => Table.dense);
 
-  const propetries = { ...rest };
-  const tableCells = Object.values(propetries).map((propetry, index) => (
-    <TableCell key={index}>{propetry}</TableCell>
+  const properties = { ...rest };
+  const tableCells = Object.values(properties).map((property, index) => (
+    <TableCell key={index}>{`${property}`}</TableCell>
```

Comment:
There is no need to do string interpolation here, you can just use {property} without any problems

Reply:
Don't use `index` as a key, use `property` instead if it is always unique and stable or find some another value for the index, please

Reply:
fixed